docs/install.md

```diff
@@ -29,6 +32,27 @@ import pydantic
 print('compiled:', pydantic.compiled)
 ```
 
+### Performance vs package size trade-off
+
+Compiled binaries can increase the size of your Python environment. If for some reason you want to reduce size the size of your *pydantic* installation you can avoid installing any binaries using the [`pip --no-binary`](https://pip.pypa.io/en/stable/reference/pip_install/#install-no-binary) option. Make sure `Cython` is not in your environment, or that you have the `SKIP_CYTHON` environment variable set to avoid re-compiling *pydantic* libraries:
```

Comment:
```suggestion
Compiled binaries can increase the size of your Python environment. If for some reason you want to reduce the size of your *pydantic* installation you can avoid installing any binaries using the [`pip --no-binary`](https://pip.pypa.io/en/stable/cli/pip_install/#install-no-binary) option. Make sure `Cython` is not in your environment, or that you have the `SKIP_CYTHON` environment variable set to avoid re-compiling *pydantic* libraries:
```

Reply:
Typo + moved link